pkg/reconcilers/base_reconciler.go

```diff
@@ -31,6 +31,44 @@ func CreateOnlyMutator(existing, desired common.KubernetesObject) (bool, error)
 	return false, nil
 }
 
+// Remove 'ThreeScaleApicastRequestTime' alert
+func RemovePrometheusRulesMutator(existing, desired common.KubernetesObject) (bool, error) {
```

Comment:
I would move this method from `base_reconciler` to `pkg/reconcilers/prometheus_rules.go`

Reply:
And add tests to it :)